packages/scripts/config/webpack.config.js

```diff
@@ -81,7 +33,17 @@ const config = {
 		filename: '[name].js',
 		path: path.resolve( process.cwd(), 'build' ),
 	},
-	externals,
+	externals: [
+		{
+			// Distributed NPM packages may depend on Babel's runtime regenerator.
+			// In a WordPress context, the regenerator is assigned to the global
+			// scope via the `wp-polyfill` script. It is reassigned here as an
+			// externals to reduce the size of generated bundles.
+			//
+			// See: https://github.com/WordPress/gutenberg/issues/13890
+			'@babel/runtime/regenerator': 'regeneratorRuntime',
```

Comment:
If I understand correctly, this should be added to resolve:

`@babel/runtime/regenerator` -> `regeneratorRuntime` (external)
and
`@babel/runtime/regenerator` -> `wp-polyfill` (script dependency)

Is that right?

https://github.com/WordPress/WordPress/blob/8ffebab3e6910123905fb238c94b7b81ef3fc1d3/wp-includes/js/dist/vendor/wp-polyfill.js#L6250

Reply:
Yes, correct. We add `wp-polyfill` by default to every `wp-*` script handle but you can do it with webpack as well, good idea 😄